src/cloud/setup/first-time-setup-import-import.md

```diff
@@ -306,6 +306,9 @@ bin/magento cache:clean
 
 After the [cache](https://glossary.magento.com/cache) flushes, enter `exit` to close the SSH tunnel.
 
+{:.bs-callout-info}
+To clear composer cache on Cloud environment use [magento-cloud project:clear-build-cache](https://devdocs.magento.com/guides/v2.4/reference/cli/magento-cloud.html) command.
```

Comment:
Please use the liquid variables for versioned devdocs URLs.

Reply:
@dobooth done, please check.

thank you!